framework/src/main/java/org/checkerframework/framework/util/JavaExpressionParseUtil.java

```diff
@@ -461,7 +536,7 @@ public JavaExpression visit(NameExpr expr, JavaExpressionContext context) {
          *
          * @param type the type in which to search for {@code identifier}
          * @param identifier possible class name
-         * @return the {@code ClassName} for {@code identifier} or null if it is not a class name
+         * @return the {@code ClassName} for {@code identifier}, or null if it is not a class name
          */
         protected @Nullable ClassName getIdentifierAsUnqualifiedClassName(
                 TypeMirror type, String identifier) {
```

Comment:
Below, there is a test
```java
            Symbol.PackageSymbol packageSymbol =
                    resolver.findPackage("java.lang", pathToCompilationUnit);
            if (packageSymbol != null) {
```

How can `packageSymbol` be null?  Should an exception be thrown in that case?